storage/tests/test_storage.py

```diff
@@ -0,0 +1,55 @@
+import random
+import string
+
+import pytest
+
+from robocorp import storage
+
+random_str = lambda length: "".join(  # noqa: E731
+    random.choice(string.ascii_lowercase) for _ in range(length)
+)
+
+
+@pytest.mark.xfail(reason="not ready for CI yet (Control Room injected env vars)")
+class TestLibrary:
```

Comment:
I copied the model from the legacy lib, where I expect to have other test cases separating concerns like: `TestRobocorpAdapter`, `TestFileAdapter` etc.

But you're on point, this is a YAGNI case, feel free to refactor.

Reply:
Thanks for the explanation... I guess I just end up creating different files instead of organizing per classes (and moving common code to a `fixtures` if needed). I guess both work ;)